install

```diff
@@ -95,6 +95,13 @@ do_install()
   # Homebrew python gets us bottles for numpy, scipy, etc.
   brew tap homebrew/python
 
+  # Sierra no longer has a working qt4 so use this private version until the issue is resolved.
+  if [[ `sw_vers -productVersion` > "10.11" ]]
+  then
+    brew install cartr/qt4/qt
+    SKIP_KEYS="--skip-keys=pyqt"
```

Comment:
I'm guessing you mean putting the brew in a rosdep.
I think I still need the SKIP_KEYS.

Would you want to switch the rosdep override based on osx version here in the script?